tests/console/apparmor.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: apparmor-utils
+# Summary: Test apparmor utilities
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use base "consoletest";
+use services::apparmor;
+use strict;
+use testapi;
+use utils;
+use warnings;
+
+sub run {
+    select_console 'root-console';
+    my $is_running = (systemctl("status apparmor", ignore_failure => 1) == 0);
+    if ($is_running) {
```

Comment:
We can avoid the variable assignment:

```suggestion
    if (systemctl("status apparmor", ignore_failure => 1) == 0) {
```

Reply:
I believe it's more readable with the variable, to make it clear that the test is only viable if the apparmor service is running. I prefer to keep it if that's OK!